tools/run_tests/artifacts/artifact_targets.py

```diff
@@ -388,10 +395,10 @@ def targets():
         PythonArtifact('manylinux2010', 'x86', 'cp37-cp37m'),
         PythonArtifact('manylinux2010', 'x86', 'cp38-cp38'),
         PythonArtifact('manylinux2010', 'x86', 'cp39-cp39'),
-        PythonArtifact('manylinux2014', 'aarch64', 'cp36-cp36m'),
-        PythonArtifact('manylinux2014', 'aarch64', 'cp37-cp37m'),
-        PythonArtifact('manylinux2014', 'aarch64', 'cp38-cp38'),
-        PythonArtifact('manylinux2014', 'aarch64', 'cp39-cp39'),
+        PythonArtifact('manylinux_2_24', 'aarch64', 'cp36-cp36m'),
```

Comment:
As you mentioned in https://github.com/grpc/grpc/issues/26279#issuecomment-852128795, 3.6 isn't impacted by this bug. Do we want to keep 3.6 wheel on manylinux2014?

Reply:
I think it's better to stay consistent and leave all the aarch64 wheels as manylinux_2_24.
- staying consistent is better
- it seems that something's wrong with libstdc++ in the dockross manylinux image and we don't know what exactly it is, and it could simply be that with python3.6 the reproduction we have happens not to crash, but things could crash under slighly different conditions. it's better to avoid that.
- specialcasing python3.6 doesn't seem to make sense since python3.6 will go end-of-life later this year.